packages/textlint-tester/test/textlint-tester-invalid-testcofig-test.js

```diff
@@ -0,0 +1,150 @@
+        expectedErrorMessage: "TestConfig is empty"
+    },
+    {
+        description: "TestConfig has no rules, plugins only",
+        config: {
+            plugins: [
+                {
+                    pluginId: "html",
+                    plugin: htmlPlugin
+                }
+            ]
+        },
+        expectedErrorMessage: "TestConfig.rules should be an array"
+    },
+    {
+        description: "TestConfig has rules property which is not an array",
+        config: {
+            rules: "rule",
+            plugins: [
+                {
+                    pluginId: "html",
+                    plugin: htmlPlugin
+                }
+            ]
+        },
+        expectedErrorMessage: "TestConfig.rules should be an array"
+    },
+    {
+        description: "TestConfig has rules property which is empty",
+        config: {
+            rules: [],
+            plugins: [
+                {
+                    pluginId: "html",
+                    plugin: htmlPlugin
+                }
+            ]
+        },
+        expectedErrorMessage: "TestConfig.rules should have at least one rule"
+    },
+    {
+        description: "One of rules has no ruleId",
+        config: {
+            rules: [
+                {
+                    rule: noTodoRule
+                }
+            ]
+        },
+        expectedErrorMessage: "ruleId property not found"
+    },
+    {
+        description: "One of rules has no rule",
+        config: {
+            rules: [
+                {
+                    ruleId: "no-todo"
+                }
+            ]
+        },
+        expectedErrorMessage: "rule property not found"
+    },
+    {
+        description: "TestConfig has plugins proeprty which is not an array",
+        config: {
+            plugins: htmlPlugin,
+            rules: [
+                {
+                    ruleId: "no-todo",
+                    rule: noTodoRule
+                }
+            ]
+        },
+        expectedErrorMessage: "TestConfig.plugins should be an array"
+    },
+    {
+        description: "One of plugins has no pluginId",
+        config: {
+            plugins: [
+                {
+                    plugin: htmlPlugin
+                }
+            ],
+            rules: [
+                {
+                    ruleId: "no-todo",
+                    rule: noTodoRule
+                }
+            ]
+        },
+        expectedErrorMessage: "pluginId property not found"
+    },
+    {
+        description: "One of plugins has no plugin",
+        config: {
+            plugins: [
+                {
+                    pluginId: "html"
+                }
+            ],
+            rules: [
+                {
+                    ruleId: "no-todo",
+                    rule: noTodoRule
+                }
+            ]
+        },
+        expectedErrorMessage: "plugin property not found"
+    }
+];
+
+describe("new-style-of-test: invalid testConfig", () => {
```

Comment:
lack of invalid case

- https://github.com/textlint/textlint/pull/491#discussion_r163429604 assert invalid `options` pattern.